### DISCUSSION TITLE
为什么把datetime控件给移除了？

### DISCUSSION THREAD

**Discussion by tornodo** [netease-im/NIM_Duilib_Framework#333] [General]
这个控件蛮常用的，如果要加入这个控件的话有实现思路可以提供的吗？

**Comment by nmgwddj** [comment]
DateTime 控件的确没有保留下来，如果迫切需要，您可以参考原版 Duilib 中，将其功能移植过来使用。或者使用前端方案，利用一些丰富的前端框架来弥补这些不足。

**Comment by tornodo** [comment]
`
if( uMsg == WM_CREATE ) {
			m_pOwner->GetManager()->AddNativeWindow(m_pOwner, m_hWnd);
			bHandled = FALSE;
		}
`
原版的datetime控件这个不知道该怎么处理，能帮帮忙吗？😄

**Comment by smallevilbeast** [comment]
![image](https://user-images.githubusercontent.com/1271747/77667950-ca320800-6fbd-11ea-9e2f-7f81302068ed.png)

项目上需要，添加了一个

**Reply by KnIfER** [reply]
厉害，这么做的，这是原版的dm吗?
